work.openpaper.Paperwork.json

```diff
@@ -0,0 +1,402 @@
+{
+    "app-id": "work.openpaper.Paperwork",
+    "branch": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "paperwork",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
```

Comment:
Wouldn't it make more sense to grant access only to pictures, documents and downloads?

Reply:
Not really. Paperwork works mostly in ~/papers by default, but user can place this work directory anywhere else (for instance, mine is ~/SparkleShare/papers). Also Paperwork provides import and export features. Those features need full access /home.

Reply:
:+1: